guides/ageur-power-usecase.md

```diff
@@ -0,0 +1,63 @@
+# Get paid in agEUR, receive funds in your bank account
+
+![Untitled](/.gitbook/assets/angle-borrowmodulewp-announcement-light.png)
+
+Thanks to its powerful partners and integrations, agEUR can now be used to pay and create invoices, and agEUR can be easily transferred to your home bank account. All of this from almost anywhere in the world, on Ethereum or Polygon!
```

Comment:
agEUR can be used to pay and create invoices.

Goal of the docs is to be permanent, not like an article, and so better not to use "now"